app/src/layouts/DashboardLayout/AppNotificationBanner/index.tsx

```diff
@@ -24,10 +29,44 @@ interface Banner {
 
```

Comment:
You also need to expand this interface accordingly.
Looking through the dummy testing banners and the following code, quickly visible suggestions are:
1. `cta` is now optional
2. `actions` does not exist right now
3. `badgeColor` is a new optional attribute

Reply:
If we are planning to work on improving this interface futher, can we rename `short_text` to `shortText`?

Reply:
Also document the relationship/dependency between `short_text` and `text` in some doc-string or comment, and any other known assumptions. Since will be the main source to understand how to structure the banner JSON